docs/README.md

```diff
@@ -239,6 +239,7 @@ file.
 | server_opts                      | Object    | -          | CMON server options.                                                    |
 | server_opts.http_accept_encoding | String    | -          | When set to "enabled" CMON honors 'Accept-Encoding'                     |
 | throttle_opts                    | Object    | -          | Restify [throttle](http://restify.com/docs/plugins-api/#throttle) opts. |
+| discover_include_stopped         | String    | false      | Set to 'true' to include stopped instances in addition to running instances. |
```

Comment:
Try to reword this a bit to get the table to line up.

Reply:
@bahamat When previewing the README it lines up correctly. Same with `FEATURE_USE_CNAGENT_COMMAND_EXECUTE` in the `sdc-cnapi` README. But I changed it anyway to make it line up when viewing the markup code.